x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -59,6 +62,7 @@ interface MLInferenceProcessorsActions {
   makeCreatePipelineRequest: typeof CreateMlInferencePipelineApiLogic.actions.makeRequest;
   makeMLModelsRequest: typeof MLModelsApiLogic.actions.makeRequest;
   makeMappingRequest: typeof MappingsApiLogic.actions.makeRequest;
+  makeSimulatePipelineRequest: typeof SimulateMlInterfacePipelineApiLogic.actions.makeRequest;
```

Comment:
Using typeof (actions) doesn't work, that type is the type of an action *creator* not the type of an *action*. You have to construct those using the `Actions<Args, Response>['makeRequest']` format. Annoying, but actually accurate and useful.

This also goes for all of the other uses in this file, can you correct those while you're at it?